web_src/js/features/repo-code.ts

```diff
@@ -96,16 +97,27 @@ function showLineButton() {
 
   createTippy(btn, {
     theme: 'menu',
-    trigger: 'click',
-    hideOnClick: true,
+    trigger: 'manual', // Use manual trigger
```

Comment:
Why it needs to use "manual"? It causes other bugs like "the menu won't be hidden when click on document".